project/config/afbini/config/search_api.index.default_content.yml

```diff
@@ -124,6 +124,8 @@ field_settings:
 
 
 
+
+
```

Comment:
Do you know where these big gaps in config files are coming from?

Reply:
It appears to be a bug on exporting with the solr config, Iv now corrected this file